### DISCUSSION TITLE
v0.6.0-alpha4

### DISCUSSION THREAD

**Discussion by kaloudis** [ZeusLN/zeus#638] [Announcements]
#### New Features:
* NFC support https://github.com/ZeusLN/zeus/pull/619
* New locales https://github.com/ZeusLN/zeus/pull/630
    Norwegian Bokmal
    Swedish
    Thai
    Ukranian
    Romanian
    Slovenian (WIP)
    Russian (WIP)
    Finnish (WIP)
    Italian (WIP)
    Vietnamese (WIP)
    Japanese (WIP)
* Component: Button https://github.com/ZeusLN/zeus/pull/613

#### New Views:
* Settings https://github.com/ZeusLN/zeus/pull/587
* Intro https://github.com/ZeusLN/zeus/pull/618

#### Enhancements:
* Layer Balances: tap to open https://github.com/ZeusLN/zeus/pull/622
* Android: update launcher images https://github.com/ZeusLN/zeus/pull/615
* Privacy: Don't save onchain addresses https://github.com/ZeusLN/zeus/pull/623 https://github.com/ZeusLN/zeus/pull/629
* Fill out error screen https://github.com/ZeusLN/zeus/pull/611

#### Bug fixes:
* c-lightning-REST fixes https://github.com/ZeusLN/zeus/pull/616 https://github.com/ZeusLN/zeus/pull/625
* ZEUS-585: Fix: Back button on Lightning Invoice screen just reloads invoice https://github.com/ZeusLN/zeus/pull/626
* ZEUS-578: Add QRCodeScanner cache https://github.com/ZeusLN/zeus/pull/627
* ZEUS-600: Fix fiat conversion for keysend payments https://github.com/ZeusLN/zeus/pull/600

#### New Features:
* LNURL-auth support for LND, c-lightning, and Eclair
* AMP support for LND v0.13+
* New Routing view and functionality
* LND: coin control for on-chain sends
* Add support for Lightning Addresses (LNURL Pay)
* Locales: Added Simplified Chinese (简化字) and added major updates to Hungarian (magyar nyelv) 
* Fetch recommended fees from Mempool.space API

#### New views:
* New homepage (swipe those balances to the right to reveal nav buttons)
* New channels view
* New activity view
* New splash page and app launcher
* New themes (and more to come)
* Updated Node Info view
* Updated Channel view
* New Coins view (WIP)

#### Enhancements:
* Improved camera
* EXPERIMENTAL: new restart Tor button
* Added slide animation to balance sliders
* Added Zeus secondary logo to QR codes
* Enhanced views for LNDHub
* Add pending to Amount component and fix some SVGs

#### Bug fixes:
* Fixed routed payment history
* Fixed false failures on c-lightning
* Android: Fix implementation picker field width
* lnurl-auth signedmsg should be strict DER
* Send: fix fiat rate conversion
* Light theme fixes
* Fix display of fee rate

[iOS TestFlight](https://testflight.apple.com/join/gpVFzEHN)

[Android APK](https://zeusln.app/zeus-v0.6.0-alpha4.apk)
[Android APK Signature](https://zeusln.app/zeus-v0.6.0-alpha4.apk.asc)
[Android APK Clearsign Signature](https://zeusln.app/zeus-v0.6.0-alpha4-signature.txt)

[PGP Key](https://zeusln.app/PGP.txt)

<hr /><em>This discussion was created from the release <a href='https://github.com/ZeusLN/zeus/releases/tag/v0.6.0-alpha4'>v0.6.0-alpha4</a>.</em>

**Comment by kokojamboo** [comment]
On Android 12/ Google Pixel 5 XL the app is not starting after update.

**Reply by kaloudis** [reply]
see https://github.com/ZeusLN/zeus/issues/644#issuecomment-983855134
